internal/host/plugin/job_set_sync.go

```diff
@@ -574,5 +547,14 @@ func (r *SetSyncJob) upsertAndCleanHosts(
 		}
 	}
 
+	// update last sync time on the sets
+	i, err := r.writer.Exec(ctx, updateSyncDataQuery, []interface{}{setIds})
```

Comment:
@talanknight lets chat about this, I was originally going to add this to the transaction for the set, but we don't actually do a transaction if there are no changes...